src/components/Layout.jsx

```diff
@@ -34,6 +56,18 @@ export function Layout({ children }) {
 					combined with other information or when used to build a profile of an
 					individual, even if that individual's name is unknown.
 				</p>
+        
+        <div className='flex w-full justify-between items-center mt-8'>
+					<p>
+						{theme !== 'dark' ? 'Light Mode' : 'Dark Mode'}
+					</p>
+					<Toggle
+						label='Switch color theme'
```

Comment:
The previously requested change would require that aria labels remain the same too. This should be changed to `Toggle light theme` or something similar.